langserver/references.go

```diff
@@ -96,9 +97,21 @@ func (h *LangHandler) handleTextDocumentReferences(ctx context.Context, conn JSO
 	} else {
 		users = h.importGraph.Search(defpkg)
 	}
+	findPackage := h.getFindPackageFunc()
 	lconf := loader.Config{
 		Fset:  fset,
 		Build: bctx,
+		FindPackage: func(bctx *build.Context, importPath, fromDir string, mode build.ImportMode) (*build.Package, error) {
+			// When importing a package, ignore any
+			// MultipleGoErrors. This occurs, e.g., when you have a
+			// main.go with "// +build ignore" that imports the
+			// non-main package in the same dir.
+			bpkg, err := findPackage(ctx, bctx, importPath, fromDir, mode)
```

Comment:
(I'd like to chat about this Mon to get your perspective, I think I'm missing exact details here / I have a few questions)